src/pages/ReportSettingsPage.js

```diff
@@ -186,39 +181,50 @@ class ReportSettingsPage extends Component {
                     </View>
                     {shouldShowRename && (
                         <View style={styles.mt4}>
-                            <Text style={[styles.formLabel]} numberOfLines={1}>
-                                {this.props.translate('newRoomPage.roomName')}
-                            </Text>
-                            <View style={[styles.flexRow, styles.mb1]}>
+                            <View style={[styles.flexRow]}>
                                 <View style={[styles.flex3]}>
-                                    <RoomNameInput
-                                        initialValue={this.state.newRoomName}
-                                        policyID={linkedWorkspace && linkedWorkspace.id}
-                                        errorText={this.state.errors.newRoomName}
-                                        onChangeText={newRoomName => this.clearErrorAndSetValue('newRoomName', newRoomName)}
-                                        disabled={shouldDisableRename}
-                                    />
+                                    {!isDefaultOrArchivedRoom ? (
```

Comment:
also NAB but the !boolean always throws my brain for a loop and makes me have to think hard about which thing is getting rendered so personally I think it's easier to reason about the code if it was
```
boolean ? renderIfTrue : renderIfFalse;
```

but not the end of the world if you want to leave it the way you have it.

Reply:
definitely agree here! was just about to make this change